doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
-   > * Despite large variances in implementation, the spectrum of networking capabilities ultimately delivered to Workloads is comparatively narrow
-   > * Standard CNTT methodology (i.e. normalize interfaces, APIs, capabilities and behaviors at the reference points) applies well to networking, helping mitigate the need to be overly prescriptive about implementation  
-   > * CNTT/OPNFV MUST provide a functional networking solution for the RI, and in support of RC  
-   > * CNTT/OPNFV potentially does NOT have to provide a production networking solution for RI  
-   > * CNTT/OPNFV is NOT planning to provide a production compute solution for RI
+•	ETSI NFV does not have a separation between HW Infrastructure Management and SW Virtualisation Management and by that the cardinality of having multiple CaaS and IaaS layers on top of a shared HW Infrastructure Layer cannot be expressed
 
+•	ETSI NFV lacks a description of the reference points in between the SW Virtualisation Layers and the HW Infrastructure Layer denoted as Vl-Ha and by that cannot express Packet Flows, Control/Status Interfaces and Management Interfaces between them
 
-<a name="5.1.1"></a>
-### 5.1.1 Executive Summary
+•	ETSI NFV model does not explicitly call out an HW Abstraction layer on top of the HW resources and by that many SW Virtualisation Layers act upon the physical HW components in a non-portable way that do not allow an HW Infrastructure to compose abstracted HW resources from pools of individual HW components
+
+•	ETSI NFV lack of spelled out HW Abstraction does not enable a clear way to abstract and describe the technical possibilities that HW layer networking separation, encapsulation, acceleration, etc. can be implemented either in a HW Infrastructure Managed SmartNIC on the Server units or in the shared underlay switching fabric i.e. on the physical Switching Units 
+
+•	ETSI NFV model does not include any reference to SDN (Software Define Network) controller(s) and relevant integration points into NFVI and MANO 
+
+•	ETSI NFV model does not have a way to enable programmable forwarding planes in the HW layer controlled from higher layers of virtualization managers, orchestrators or Network Functions
+
+<p align="center"><img src="../tech/figures/ETSI.png"><b> Figure 5-1-1-1: ETSI model and Networking.</b></p>
+
+Note: Other areas of concern may be identified and addressed during the later stages of work on the RM/RA contents for Networking.
+
+<a name="5.1.1.2"></a>
+#### 5.1.1.2 Specific topics to be included in the Networking Scope
+To address the concern areas detailed above the specific topics listed below will be covered under Networking. The list below does not pretend to be complete. It will be a subject to modifications as a result of the community discussion.
+
+•	Network connectivity for L3 tenants, GWs, SDS, etc
+
+•	Spine-Leaf Architecture ( 3 Tier or 5 Tier CLOS architecture )
+
+•	Automation of the programmable newtorking fabric
```

Comment:
Definitely automation of the fabric, but whether that fabric is distributed or centralized, shouldn't impact the CNTT architecture or RM.

Reply:
@markshostak Agree and that is also discussed in chapter "5.1.3.4 Programmable Networking Fabric concept" with the following paragraphs:
"The logically centralised control plane is the foundation for the authoritative separation between different SW Virtualisation domains or Bare Metal Network Function applications that are regarded as untrusted both from the shared layers and each other.
Although the control plane is logically centralized, scaling and control latency concerns must allow the actual implementation of the control plane to be distributed when required."

Reply:
@TFredberg Differentiating between centralized and logically centralized is splitting hairs a bit. There will always be a local CAM, whether it's populated from a logically centralized executive, or by local intelligence (i.e. a stand-alone distributed switch). However, to be more clear, I should have simply said Automation of the fabric is an objective of CNTT, be it distributed or centralized, hence, we don't need to qualify it as programmable.

Reply:
@markshostak The section on Programmable Networking Fabric points out the necessity of a logically centralized control plane since the it is an RM foundation that it is just one and it is centralized. 
The section also points out that it is important that the logically centralized control plane RM definition must allow for implementation of centralized controllers and distributed controllers since both of these exist in the market. 
I have though noticed that many people in the industry tend to equate a statement of “centralized fabric controller” as identical to an SDN controller instance that is centralized by running outside of the switches that it controls. 
By this sentence I wanted to proactively clear out any possibilities of such misunderstandings.

Reply:
@TFredberg I think we should separate this into 2 threads:
1) WRT the language in question, I'm merely saying remove the word "programmable", as the automation applies irrespective of the fabric implementation.

2) WRT clarifying centralized control, I can't tell if we're fully in sync yet. When you say:
>
> I have though noticed that many people in the industry tend to equate a statement of “centralized fabric controller” as identical to an SDN controller instance that is centralized by running outside of the switches that it controls.

The doc currently defines 2 control planes (SDNu and SDNo). I proposed a third (currently SDNp, but I'm not convinced that's the best name, yet), to explicitly differentiate what I think you're trying to clarify. The first 2 are common to programmable and "non-programmable" fabrics. The third is unique to fabrics utilizing a "centralized control" to populate the elements' CAM/FDB via an API such as OF, P4, etc.

Reply:
Hi, Who wrote the "dears , what do you mean by programmable and..." comment? For some reason GH is showing it as if it's coming from me (Mark).